activerecord/lib/rails/generators/active_record/model/model_generator.rb

```diff
@@ -58,6 +60,16 @@ def parent_class_name
           end
         end
 
+        def ensure_migration_source_paths
```

Comment:
This is pretty much reimplementing [`find_in_source_paths`](https://github.com/rails/thor/blob/376e141adb594f3146c57e98151b97a20c93c484/lib/thor/actions.rb#L133).

Can't we just use it?

Reply:
Thanks for the review @rafaelfranca ! `find_in_source_paths` is being called by [migration_template](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/migration.rb#L57) so we are using it. 

The issue this PR is solving is that `source_paths` do not contain the correct paths for customizable `create_table_migration.rb.tt` templates. So the solution is to include the template_paths from the migration generator, as well a "special" path to preserve existing functionality. 

I'm open to any feedback to get this functionality fixed 😃

Reply:
[`Rails::Generators.templates_path` should already be in the source_paths](https://github.com/rails/rails/blob/99096b88ae7a7c9019f2cbcf867e29ba42768e65/railties/lib/rails/generators/base.rb#L242-L259). We don't do any of this work in other generators and they just work.

I think the source of the problem is that we are using relative path to find the template. What we can do is append the migration template to the source path in this generator and change back the relative path.